manifests/config.pp

```diff
@@ -6,14 +6,24 @@
 
   assert_private()
 
-  file { $::telegraf::config_file:
-    ensure  => file,
-    content => template('telegraf/telegraf.conf.erb'),
-    mode    => '0640',
-    owner   => 'telegraf',
-    group   => 'telegraf',
-    notify  => Class['::telegraf::service'],
-    require => Class['::telegraf::install'],
+  file {
```

Comment:
Updated. I've repeated the parameters on each file than using `File {}` to avoid scope-related weirdness, however unlikely.